cmd/podman/cp.go

```diff
@@ -95,7 +101,7 @@ func copyBetweenHostAndContainer(runtime *libpod.Runtime, src string, dest strin
 	if err != nil {
 		return err
 	}
-	defer ctr.Unmount(false)
+	defer silentUnmount(ctr, false)
```

Comment:
Nit: since that's the only caller, we could also do a:
```golang
defer func() {
        if err := ctr.Unmount(false); err != nil {
		logrus.Errorf("unable to umount container '%s': %q", ctr.ID(), err)
	}
}()
```

Reply:
agree